include/irr/asset/SSamplerParams.h

```diff
@@ -36,7 +36,7 @@ enum E_TEXTURE_BORDER_COLOR
 enum E_TEXTURE_FILTER
```

Comment:
those enums should really be defined inside `ISampler`

Reply:
resolved